app/controllers/incident/location/UnLocodeController.scala

```diff
@@ -65,11 +65,11 @@ class UnLocodeController @Inject() (
         .fold(
           formWithErrors => Future.successful(BadRequest(view(formWithErrors, mrn, mode, index))),
           value =>
-            unLocodesService.doesUnLocodeExist(value).flatMap {
+            unLocodesService.doesUnLocodeExist(value.toUpperCase).flatMap {
               case true =>
                 implicit val navigator: UserAnswersNavigator = navigatorProvider(mode, index)
                 UnLocodePage(index)
-                  .writeToUserAnswers(value)
+                  .writeToUserAnswers(value.toUpperCase)
```

Comment:
Consider handling this in the form provider with
`"value" -> adaptedText(s"$prefix.error.required")(_.toUpperCase)`